client/payment-methods/index.js

```diff
@@ -341,6 +365,13 @@ const PaymentMethods = () => {
 						) }
 					</PaymentMethodsList>
 				</CardBody>
+			</Card>
+			<br />
```

Comment:
It looks like this `<br />` tag is causing an unwanted "line"/border once the UPE has been enabled

![Screenshot 2023-09-14 at 12 52 24 PM](https://github.com/Automattic/woocommerce-payments/assets/273592/afa40d67-57cb-4eaf-b3c9-019bc121bb19)

Reply:
Great catch and perfect suggestion. Fixed in 37c362d90bb4aed747ea82b099e4f77ae125c3cc